cpp/oneapi/dal/algo/kmeans_init/backend/cpu/compute_kernel_dense.cpp

```diff
@@ -43,9 +45,15 @@ static compute_result<Task> call_daal_kernel(const context_cpu& ctx,
     const std::int64_t column_count = data.get_column_count();
     const std::int64_t cluster_count = desc.get_cluster_count();
 
+    const std::int64_t init_trial_count = desc.get_local_trials_count();
+    const auto additional = std::log(cluster_count);
```

Comment:
just a suggestion: probably its better to replace with

```
std::int64_t trial_count = desc.get_local_trials_count();
if(trial_count== -1)
{
const auto additional = std::log(cluster_count);
trial_count = 2 + std::int64_t(additional);
}
```
to avoid additional computations in case when trial count is set